### ISSUE TITLE
Simple version bump

### ISSUE THREAD

**Issue by khjohnson1972** [pixelcog/parallax.js#118]
Can you bump the versions in parallax.js doc block. Was confused when installing using bower and reviewing parallax.js.

The doc block reads 1.3.1 on 1.3.2 and 1.3.1 on 1.4.0:

```
/*!
 * parallax.js v1.3.1 (http://pixelcog.github.io/parallax.js/)
```

Thanks -

**Comment by mikegreiling** [comment]
yep, sorry about that...  I added another guy with push access to the repository and he's still learning.  I'll get it fixed for the next version bump.

**Comment by mikegreiling** [comment]
should be fixed in 9ed1680
